modules/saml/src/Auth/Source/SP.php

```diff
@@ -868,7 +888,7 @@ public function reauthenticate(array &$state): void
          * original IdP used a compatible AuthnContext. If not, we may need
          * need to do step-up authentication (e.g. for requesting MFA). At
          * the moment this only handles exact comparisons, since handling
-         * better/minimum/maximum would require a prioritised list.
+         * better/minimum/maximum would require a prioritized list.
```

Comment:
I think this is just US spelling vs UK spelling. I'm not sure if it is worth changing.

Reply:
done